Sources/SwiftSyntax/SyntaxParser.swift

```diff
@@ -158,8 +158,9 @@ func makeRawToken(_ c_node: CSyntaxNode, source: String) -> RawSyntax {
   }
 
   let offset = Int(c_node.range.offset)
+  let totalLen = Int(c_node.range.length)
   let tokOffset = offset + leadingTriviaLen
-  let tokLen = Int(c_node.range.length) - leadingTriviaLen - trailingTriviaLen
+  let tokLen = totalLen - leadingTriviaLen - trailingTriviaLen
```

Comment:
Ditto on subtraction associativity here.
```suggestion
  let tokLen = totalLen - (leadingTriviaLen + trailingTriviaLen)
```